bindings/rust/src/lib.rs

```diff
@@ -453,6 +513,15 @@ impl Instance {
 mod tests {
     use super::*;
 
+    #[test]
+    fn error_box() {
+        let mut err = FizzyErrorBox::new();
+        assert_ne!(unsafe { err.as_mut_ptr() }, std::ptr::null_mut());
+        assert_eq!(err.code(), 0);
+        assert_eq!(err.message(), "");
+        assert_eq!(format!("{}", err), "0 []");
```

Comment:
Is it possible to have a test also for `format!`  of failure?

Reply:
Not really. That would mean doing some pointer magic to set the contents.